src/spectrecoff/Spectre/Progress.fs

```diff
@@ -44,7 +44,12 @@ let defaultTemplate =
     |> withPercentageColumn
 
 let startCustom template (operation: ProgressContext -> Task<unit>) =
```

Comment:
We could define a type alias for this.

Reply:
We could, but I saw that the type can be inferred after all, so I just removed it. Better no?

Reply:
edit: it could not be inferred after all. Added type alias for this and for status as you suggested.